integration-tests/multitenancy.test.ts

```diff
@@ -63,6 +63,59 @@ if (process.env.MULTI_TENANCY_ENABLED === 'true') {
             );
         });
 
+        test('tenant cannot SEARCH _include or _revinclude resources from another tenant', async () => {
+            const testOrganization = {
+                resourceType: 'Organization',
+                name: 'Some Organization',
+            };
+
+            const testOrganizationResource = (await client.post('Organization', testOrganization)).data;
+
+            const testPatientWithRelativeReferenceToOrg: ReturnType<typeof randomPatient> = (
+                await clientForAnotherTenant.post('Patient', {
+                    ...randomPatient(),
+                    managingOrganization: {
+                        reference: `Organization/${testOrganizationResource.id}`,
+                    },
+                })
+            ).data;
+
+            const testPatientWithAbsoluteReferenceToOrg: ReturnType<typeof randomPatient> = (
+                await clientForAnotherTenant.post('Patient', {
+                    ...randomPatient(),
+                    managingOrganization: {
+                        reference: `${process.env.API_URL}/tenant/tenant1/Organization/${testOrganizationResource.id}`,
+                    },
+                })
+            ).data;
+
+            await waitForResourceToBeSearchable(clientForAnotherTenant, testPatientWithAbsoluteReferenceToOrg);
+
+            await expectResourceToNotBePartOfSearchResults(
+                clientForAnotherTenant,
+                { url: 'Patient', params: { _id: testPatientWithRelativeReferenceToOrg.id, _include: '*' } },
```

Comment:
I don't think it would be an issue since we are passing in `_id`, but is it possible for these include/revincludes reaching page limits? i.e. the actual result is on a future page?

Reply:
That cannot happen. 

We are creating a new Patient specifically for this test. The search response will have exactly one resource(The Patient with the provided `_id`, and no other resources since the include/revinclude * should not match anything)